Utilities.Rest/Packages/com.utilities.rest/Runtime/Interfaces/IDownloadCache.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Utilities.WebRequestRest.Interfaces
+{
+    public interface IDownloadCache
+    {
+        void ValidateCacheDirectory();
+        Task ValidateCacheDirectoryAsync();
+        bool TryGetDownloadCacheItem(string uri, out string filePath);
+        bool TryDeleteCacheItem(string uri);
+        void DeleteDownloadCache();
+        Task CacheItemAsync(byte[] data, string cachePath, CancellationToken cancellationToken);
```

Comment:
I'm wondering if there's a way we can do this without making it public. I don't like that a consumer of package can save to the cache, since the internals should be abstracted away from them, and they should only be interacting with the Rest class only.

Reply:
Ahh, I see you fixed that by making the IDownloadCache "internal"